<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="pro_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>生产者</string>
    </property>
   </widget>
   <widget class="QLabel" name="con_1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>消费者 #1</string>
    </property>
   </widget>
   <widget class="QLabel" name="con_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>消费者 #2</string>
    </property>
   </widget>
   <widget class="QLabel" name="con_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>消费者 #3</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pro1_text">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>280</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="con1_text">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>280</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="con2_text">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>280</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="con3_text">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>70</y>
      <width>280</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>84</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>停止（清空队列）</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
